### PR TITLE
Add non-color bulb to tested devices

### PR DESCRIPTION
**Pull request by parliament119**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/89141
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/fritzbox.markdown [hunk 1]
```diff
@@ -54,6 +54,7 @@ Additionally, we also support to trigger smarthome templates.
 - [FRITZ!DECT 500](https://en.avm.de/products/fritzdect/fritzdect-500/)
 - [Eurotronic Comet DECT](https://eurotronic.org/produkte/dect-ule-heizkoerperthermostat/comet-dect/)
 - [Magenta SmartHome LED E27](https://www.smarthome.de/geraete/smarthome-led-lampe-e27-farbig-weiss)
+- Magenta SmartHome LED E27 warmwhite
```
Comment (mib1185) [inline source/_integrations/fritzbox.markdown:57]:
```suggestion
- [Magenta SmartHome LED E27 warmwhite](https://www.smarthome.de/geraete/smarthome-wlan-led-lampe-e27-weiss)
```

**Review by mib1185 — CHANGES_REQUESTED** [review]
hi @parliament119 please also add the link to product description

File: source/_integrations/fritzbox.markdown (hunk 1 above)
Comment (mib1185) [reply source/_integrations/fritzbox.markdown:57]: also please adjust the name above to indicate it is the colored bulb

Comment (parliament119) [reply source/_integrations/fritzbox.markdown:57]: I added the color info to the existing tested bulb.

Unfortunately they don't seem to sell the warmwhite lamp anymore. The one i have uses the DECT-ULE HANFUN protocol for communicaiton, not wifi. I also couldn't find any other link.

Comment (mib1185) [reply source/_integrations/fritzbox.markdown:57]: oh ... you are right, this is a WiFi bulp, not dect

**Review by mib1185 — APPROVED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @parliament119 👍 

../Frenck

**Pull request merged**